images/php/Dockerfile.cli

```diff
@@ -1,6 +1,6 @@
 ARG PHP_VERSION=8.3
 FROM php:${PHP_VERSION}-cli-alpine AS php-cli
-FROM ghcr.io/skpr/mtk:latest AS mtk
+FROM ghcr.io/skpr/mtk:v2.0.2 AS mtk
```

Comment:
@nicksantamaria The `latest` tag has been dropped from the registry. This is the latest "stable" tag, there is a [more recent tag](https://github.com/orgs/skpr/packages/container/mtk/249077404?tag=v2.0.2-44-gf99e041) but it looks like it's related to [a PR](https://github.com/skpr/mtk/pull/29)

Reply:
I'm fine with this 👍